packages/@react-aria/tag/src/useTagGroup.ts

```diff
@@ -24,6 +24,18 @@ export interface TagGroupAria {
   tagGroupProps: DOMAttributes
 }
 
+export interface AriaTagGroupProps<T> extends TagGroupProps<T>, DOMProps, AriaLabelingProps {
+  /**
+   * An optional keyboard delegate to handle arrow key navigation,
+   * to override the default.
+   */
+  keyboardDelegate?: TagKeyboardDelegate<T>,
+  /**
+   * Whether navigation through tab key is enabled.
+   */
+  allowsTabNavigation?: boolean
+}
```

Comment:
Do we need to expose these in the public API? Maybe if we move the buttons out of the grid `allowsTabNavigation` wouldn't be necessary. But also maybe you could just always set it to true? Not sure if we would ever have tabbable elements inside tags.

Reply:
Agreed, I'll remove this if I move the button outside of the grid and fix the layout. If we can't do that, I can just set this to true.

Reply:
Removed this since the button is now outside of the grid.